thumbor/context.py

```diff
@@ -67,11 +35,16 @@ def __init__(self, server=None, config=None, importer=None, request_handler=None
         self.config = config
         if importer:
             self.modules = ContextImporter(self, importer)
+            if importer.metrics:
+                self.metrics = importer.metrics(config)
+            else:
+                self.metrics = Metrics(config)
         else:
             self.modules = None
+            self.metrics = Metrics(config)
```

Comment:
this initialization logic feels a bit clumsy, but i'm not sure how the `ContextImporter` is supposed to be used otherwise. the problem here is that using `context.modules.metrics` would not work very well as `context.metrics` is called directly, there is no check if it's set or not.